dist-persist/wbstack/src/Internal/PreApiWbStackUpdate.php

```diff
@@ -16,9 +16,38 @@ public function execute() {
 		@ini_set( 'memory_limit', '-1' ); // also try to disable the memory limit? Is this even a good idea?
 
 		// Run update.php
+		$domain = $GLOBALS[WBSTACK_INFO_GLOBAL]->requestDomain;
 		$mwPath = realpath( __DIR__ . '/../../../' );
-		$cmd = 'WBS_DOMAIN=' . $GLOBALS[WBSTACK_INFO_GLOBAL]->requestDomain . ' php ' . $mwPath . '/maintenance/update.php --quick';
-		exec($cmd, $out, $return);
+		$cmd = 'WBS_DOMAIN=' . $domain . ' php ' . $mwPath . '/maintenance/update.php --quick 2>&1';
+
+		$stdout = fopen( 'php://stdout', 'w' );
```

Comment:
This is opened, but never closed. I suggest adding `fclose($stdout)` at the end to close this.